src/main/java/io/fabric8/maven/docker/util/EnvUtil.java

```diff
@@ -392,4 +392,32 @@ public static Date loadTimestamp(File tsFile) throws MojoExecutionException {
     public static boolean isWindows() {
         return System.getProperty("os.name").toLowerCase().contains("windows");
     }
+    
+    public static boolean isValidWindowsFileName(String text) {
+    	
+    	if (!isWindows()) {
```

Comment:
Please put the windows checks outside this method (e.g. where it is called) as otherwise the semantics of this check is confusing (e.g. returning different return values depending on the same argument, depending on on which OS the method is called)